e2e/README.md

```diff
@@ -0,0 +1,6 @@
+## End-to-End tests
+
+End-to-End tests require running backend and frontend. To run tests type `npm start` and choose Specification file with
+defined tests.
+
+To learn about best practices see[https://docs.cypress.io/guides/references/best-practices.html](https://docs.cypress.io/guides/references/best-practices.html)
```

Comment:
`...best practices go [here](https://docs.cypress.io/guides/references/best-practices.html)` or something along these lines to hide this long URL.

Reply:
Or at least add a space after `see` :)